Update _index.de.md

CMSworker opened the pull request:
Vorschlag Grammatikänderung (Dativ-Genitiv) - "dank dem Manager" ließt sich IMHO holpriger als "dank des Managers"

lionel-m commented:
@leofeyer, @aschempp : Could you merge this PR please. I would like to delete my fork and recreate a new one. Thx.

aschempp commented:
Would this be the manual for Contao 3.1? Or should it be merged into another version?

lionel-m commented:
3.2.
I forgot to specify the version. Thanks !

aschempp commented:
I have merged it into the 3.2 branch. It's obviously not yet published on the Contao website because I still need to find time to improve the documentation system. But I guess it will help you re-create your branches :-)

lionel-m commented:
Thx.

PS: At the moment the version 2 of [GitBook is under development](https://github.com/GitbookIO/gitbook/pull/562). An occasion to re-evaluate this service considering the new improvements... (reference issue #174)

lionel-m commented:
> This pull request is closed, but the lionel-m:french-translation branch has unmerged commits.

This means that some commits were ignored ?

aschempp commented:
I don't think so. I merged the branch into Contao 3.2 even though it was based on 3.1, so maybe that was the problem…

CMSworker committed:
Update _index.de.md

Vorschlag Grammatikänderung (Dativ-Genitiv) - "dank dem Manager" ließt sich IMHO holpriger als "dank des Managers"

netzarbeiter commented:
Thank you @CMSworker
